src/systems/systemDetails.component.tsx

```diff
@@ -66,27 +64,33 @@ function SystemDetails(props: SystemDetailsProps) {
     </Box>
   ) : (
     <>
-      <Box
+      <Grid
+        container
+        item
         sx={{
           display: 'flex',
           alignItems: 'center',
-          margin: 1.5,
+          my: '5px',
```

Comment:
```suggestion
          my: 1.25,
```

Reply:
The suggestion is wrong it would be 0.625